dockerfiles/build.sh

```diff
@@ -9,6 +9,8 @@
 #
 # See: https://sipb.mit.edu/doc/safe-shell/
 
+set -e
```

Comment:
It's had been fixed in other repositories

```
base_dir=$(cd "$(dirname "$0")"; pwd)
. "${base_dir}"/build.include
```

Reply:
"set -e" wouldn't hurt though

Reply:
you can keep it, but if we include absolute path, first instruction of build.include is `set -e`